muxsearch/main.c

```diff
@@ -175,15 +175,17 @@ void gen_label(char *item_glyph, char *item_text, char *item_data) {
     lv_obj_t *ui_lblResultItemGlyph = lv_img_create(ui_pnlResult);
     apply_theme_list_glyph(&theme, ui_lblResultItemGlyph, mux_module, item_glyph);
 
-    lv_group_add_obj(ui_group, ui_lblResultItem);
-    lv_group_add_obj(ui_group_glyph, ui_lblResultItemGlyph);
-    lv_group_add_obj(ui_group_panel, ui_pnlResult);
-
     apply_size_to_content(&theme, ui_pnlContent, ui_lblResultItem, ui_lblResultItemGlyph, item_text);
     apply_text_long_dot(&theme, ui_pnlContent, ui_lblResultItem, item_text);
 
     lv_obj_set_user_data(ui_lblResultItem, item_data);
+    lv_obj_set_user_data(ui_pnlResult, (void*)(intptr_t)ui_count);
 
+    if (strcasecmp(item_data, "blank") != 0 && strcasecmp(item_data, "folder") != 0) {
```

Comment:
by excluding the objects from the navigation groups LVGL will auto skip blank entries and folder entries